assemble/conf/templates/generic_logger.xml

```diff
@@ -25,7 +25,7 @@
      <param name="MaxBackupIndex" value="10"/>
      <param name="Threshold"      value="DEBUG"/>
      <layout class="org.apache.log4j.PatternLayout">
-       <param name="ConversionPattern" value="%d{ISO8601} [%-8c{2}] %-5p: %m%n"/>
+       <param name="ConversionPattern" value="%d{ISO8601} [%-8c{2}] [pid:%6X{pid}] %-5p: %m%n"/>
```

Comment:
> If you run 3 RS on a host, the first started RS would have a 1 in the log name, a 2 in the second, 3 in the third

That would be nice, just have to do some IPC on the node for a process to reserve number.  Once a process reserves a number it can roll and age off logs for that number.